approaching_emergency_vehicle_plugin/config/parameters.yaml

```diff
@@ -37,6 +37,11 @@ minimum_reduced_speed_limit : 2.2352 # NOTE: 2.2352 m/s is 5 mph
 # Units: m/s
 default_speed_limit : 2.2352 # NOTE: 2.2352 m/s is 5 mph
 
+# Double: A buffer value; if the ego vehicle speed is within this speed of its target speed when slowing down for an actively passing
```

Comment:
ego or CMV?

Reply:
I'm using "ego vehicle" as a generic term to used to refer to the host connected and/or automated vehicle. I used "ego vehicle" (rather than "CMV") in all comments in this plugin to refer to the host vehicle equipped with the CARMA System, since the plugin could be activated on non-commercial motor vehicles.

Reply:
Was there any initial definition of this ego somewhere in this plugin that is not in this PR before we refer to ego?

Reply:
I haven't included a definition, but I believe "ego vehicle" is a common term used for referring to a host connected/automated vehicle, and is used in other places within carma-platform and our Confluence documentation as well.